sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/ChangeFeedPartitionReader.scala

```diff
@@ -83,8 +83,12 @@ private case class ChangeFeedPartitionReader
       cosmosChangeFeedConfig.changeFeedMode match {
           case ChangeFeedModes.Incremental =>
```

Comment:
Maybe combine the two cases into:
`case ChangeFeedModes.Incremental | ChangeFeedModes.LatestVersion`